components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -335,9 +347,56 @@ - (void)addContentViewController:(UIViewController *)viewController {
   if (viewController && doesNotContainViewController) {
     [self addChildViewController:viewController];
     [self.content addSubview:viewController.view];
-    [self addConstraintsForContentView:viewController.view];
+    [self addConstraintsForChildViewControllerView:viewController.view];
     [viewController didMoveToParentViewController:self];
   }
+  [self updateNavigationBarInsets];
+}
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  [self updateNavigationBarInsets];
```

Comment:
why are we updating the insets in both viewDidAppear and viewDidLayoutSubviews, wouldn't just putting it here suffice?

Reply:
Yes, it looks like this is sufficient.